fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Encoding.kt

```diff
@@ -70,7 +74,8 @@ class Encoding(val httpMethod: Method,
 
     private fun queryFromParameters(params: List<Pair<String, Any?>>?): String = params.orEmpty()
             .filterNot { it.second == null }
-            .joinToString("&") { "${URLEncoder.encode(it.first,"UTF-8")}=${URLEncoder.encode("${it.second}","UTF-8")}"}
+            .map { (key, value) ->  URLEncoder.encode(key, "UTF-8") to URLEncoder.encode("$value", "UTF-8") }
```

Comment:
Just `value` maybe? 😄

Reply:
param list is of type Pair<String,Any> and URLEncoder expects parameter to be of type String

Reply:
Ah ... you are right. Thanks!